### PR TITLE
ConnectWallet v3 

### PR DESCRIPTION
**Pull request by MananTank**

### CODE REVIEW

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**…hboard/wallets/connect**](https://thirdweb-8aoklug24.thirdweb-preview.com/dashboard/wallets/connect) |
|:-:|
|<a href="https://thirdweb-8aoklug24.thirdweb-preview.com/dashboard/wallets/connect">Screenshot of /dashboard/wallets/connect</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/nWgXCQzkQ1TvEB_UVPmd8Fc4tkrxRanrRfydUm57tX_-BfZyqGP0n_0hZEUetbOQFueBkGqTt7cyrYxhGIuVIw.png">(view full size)</a> |







Commit 8cba5224bca79da614fb24eeaed279b1a4fc3f57 (https://thirdweb-8aoklug24.thirdweb-preview.com).

**Comment by MananTank** [comment]
something is up with IPFS gateway - getting build errors

File: components/wallets/ConnectWalletWithPreview.tsx [hunk 1]
```diff
@@ -98,13 +102,16 @@ const trustWalletConfig = trustWallet();
...
+const phantomConfig = phantomWallet();
 
 const hideUIForWalletIds = new Set([
   metamaskWalletConfig.id,
   coinbaseWalletConfig.id,
   walletConnectConfig.id,
   bloctoWalletConfig.id,
   frameWalletConfig.id,
+  phantomConfig.id,
+  // TODO: add phantom too
```
Comment (nachoiacovino) [inline components/wallets/ConnectWalletWithPreview.tsx:114]: isn't phantom already there?

File: components/wallets/ConnectWalletWithPreview.tsx [hunk 2]
```diff
@@ -479,7 +544,7 @@ export const ConnectWalletWithPreview: React.FC = () => {
                 </FormItem>
 
                 {/* dropdownPosition */}
-                <FormItem
+                {/* <FormItem
```
Comment (nachoiacovino) [inline components/wallets/ConnectWalletWithPreview.tsx:547]: let's remove this?

**Pull request merged**